### ISSUE TITLE
MMA7660FC datasheet 4k7 pull-up resistors for I2C

### ISSUE THREAD

**Issue by gentunian** [wickerbox/Basic-Breakout-Boards#1]
Hi,

MMA7660FC datasheet recommends 4k7 pull-up resistors
![selection_080](https://user-images.githubusercontent.com/1090113/42142068-bbdfa326-7d83-11e8-9786-22a162594921.png)

Is there any reason to use 1K instead of 4k7?

Cheers!

**Comment by wicker** [comment]
Good catch! In my case, I think I had internal pullups on the MCU that I was using for testing (or I was just using a ton of 1K and no 4K7 in that design) so I'll leave this open and adjust the design to call out the 4K7 resistors from the datasheet.
